compiler/pash_runtime.py

```diff
@@ -68,6 +68,42 @@ def parse_args():
     args = parser.parse_args()
     return args
 
+
+def compile_ir(ir_filename, compiled_script_file, args):
```

Comment:
If possible, could this and the following function use the same code (instead of being copied)?

Reply:
fixed